File: table.py
# Pide al usuario el número a multiplicar
num = int(input("Dime el numero a multiplicar: "))

# Pide al usuario el límite de la tabla
limite = int(input("Dime hasta qué número quieres multiplicar: "))

# Imprime la tabla de multiplicar hasta el límite especificado
for i in range(1, limite + 1):
    print(f"{num} x {i} = {num * i}")

File: table.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int num = get_int("Dime el limite a multiplicar: ");
    int limite = get_int("Dime hasta qué número quieres multiplicar: ");

    for(int i=0; i<limite; i++)
    {
        printf("%i x %i = %i\n", num, i+1, num*(i+1));
    }
}
